core/controllers/reader.py

```diff
@@ -111,30 +111,65 @@ def get(self, exploration_id):
 class ExplorationPage(base.BaseHandler):
     """Page describing a single exploration."""
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'exploration_id': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'v': {
+                'schema': {
+                    'type': 'int'
+                },
+                'default_value': None
+            },
+            'parent': {
+                'schema': {
+                    'type': 'basestring'
+                },
```

Comment:
Any other rules for this?

Reply:
Not able to find anything extra, kindly help.
One more point I have to discuss that since 'parent' and 'iframed' are request params so they are of string type, so I can add choices field in the schema: ```['true', 'false']```
Looking forward for your point.

Reply:
Ok, regarding parent same as above.

Reply:
Added logic to normalize the string into bool by schema validation system, please take a look then I will assign @seanlip for codeowner review.